cmd/nitro/nitro.go

```diff
@@ -297,7 +298,12 @@ func main() {
 		}
 	}
 
-	currentNode, err := arbnode.CreateNode(stack, chainDb, &nodeConfig.Node, l2BlockChain, l1Client, &rollupAddrs, l1TransactionOpts)
+	var daSigner das.DasSigner
+	privateKey, err := crypto.HexToECDSA(l1Wallet.PrivateKey)
+	if err == nil {
+		daSigner = das.DasSignerFromPrivateKey(privateKey)
+	}
```

Comment:
Ignoring errors here seems odd. Instead we could skip this if the private key is an empty string and fail on error